core/dbt/include/global_project/macros/schema_tests/not_null.sql

```diff
@@ -4,8 +4,8 @@
 {% set column_name = kwargs.get('column_name', kwargs.get('arg')) %}
 
 select count(*)
-from {{ model }}
-where {{ column_name }} is null
+from {{ model }} AS aliased
```

Comment:
I just thought of something really critical here - you can currently supply an expression to these tests, eg:

```
models:
 - name: my_model
    tests:
      - unique:
          arg: concat(field_1, field_2)
```

I think code like this will no longer work if we purse this approach :/

Maybe an alternative idea is to do something like:
```
select count(*)
from (
  select {{ column_name }} as __dbt_not_null_field from {{ model }}
) as __dbt_test_sbq
where __dbt_not_null_field is null
```

I think this should support existing use cases while also addressing the current issue with BigQuery table names masking column names.